client.go

```diff
@@ -26,7 +26,7 @@ type GraphQLClient struct {
 type ClientOpt func(*GraphQLClient)
 
 // NewClient creates a new GraphQLClient from the given options.
-func NewClient(opts ...ClientOpt) *GraphQLClient {
+func NewClient(plugins []Plugin, opts ...ClientOpt) *GraphQLClient {
```

Comment:
Do we care about this being a breaking change for current `NewClient` consumers?

Reply:
@nmaquet I added [a1e312a](https://github.com/movio/bramble/pull/152/commits/a1e312a1e65e0f89fae200d6422e3e1e9f206b4b) which makes the plugin constructor a new function instead of changing the `NewClient` signature.

Reply:
Perfect, thanks.